django/db/backends/sqlite3/operations.py

```diff
@@ -193,13 +193,14 @@ def convert_uuidfield_value(self, value, expression, connection, context):
             value = uuid.UUID(value)
         return value
 
-    def bulk_insert_sql(self, fields, num_values):
-        res = []
-        res.append("SELECT %s" % ", ".join(
-            "%%s AS %s" % self.quote_name(f.column) for f in fields
-        ))
-        res.extend(["UNION ALL SELECT %s" % ", ".join(["%s"] * len(fields))] * (num_values - 1))
-        return " ".join(res)
+    def bulk_insert_sql(self, fields, placeholder_rows):
+        return " UNION ALL ".join(
+            "SELECT %s" % ", ".join(
+                p if not_first else '%s AS %s' % (p, self.quote_name(f.column))
+                for p, f in zip(row, fields)
+            )
+            for not_first, row in enumerate(placeholder_rows)
```

Comment:
I think we can actually just remove the `AS` logic. I can't find any reference to SQLite ever requiring column aliases in these circumstances, and the tests work fine without it. It's been in since the first cut: 7deb25b8dd5aa1ed02b5e30cbc67cd1fb0c3d6e6

Alternatively, SQLite supports multiple comma-separated rows in its `VALUES` clause since version 3.7.11, released March 2012. But that's probably a bit too recent to adopt: http://www.sqlite.org/changes.html#version_3_7_11